src/globals/scss/_deprecate.scss

```diff
@@ -2,19 +2,23 @@
 /// no longer going to be present in the next major release of Carbon.
 /// @access public
 /// @param {String} $reason - The message
-/// @param {Bool} $condition [true] - `true` to emits the given deprecation messsage
+/// @param {Bool} $condition [true] - `true` to emit the given deprecation messsage
+/// @param {Bool} $remove-deprecated [false] - `true` to omit the content if $condition is `true`
 /// @require {Bool} $deprecations--entry
 /// @require {Map} $deprecations--reasons
-@mixin deprecate($reason, $condition: true) {
+@mixin deprecate($reason, $condition: true, $remove-deprecated: false) {
```

Comment:
Do we need both condition and `remove-deprecated`?

Reply:
Yes, for "deprecate but not remove yet".